modules/library/src/test/scala/zio/elasticsearch/ElasticQuerySpec.scala

```diff
@@ -1074,6 +1074,70 @@ object ElasticQuerySpec extends ZIOSpecDefault {
             )
           )
         },
+        test("intervalsQuery") {
+          val intervalNoOptions = intervalMatch("lambda works")
+
+          val intervalWithOptions = intervalMatch("lambda works")
+            .withOrdered(true)
+            .withMaxGaps(2)
+            .withAnalyzer("standard")
+
+          val queryWithStringField = intervals("content", intervalWithOptions)
+
+          val typedField: Field[TestDocument, String] = Field(None, "content")
```

Comment:
Can you use `TestDocument.intField` or `stringField` instead of `typedField`? It looks okay, but also, this should be part of `ElasticIntervalQuerySpec`.